### PR TITLE
docs(OIDC): support auth methods to get token by their spec name

### PR DESCRIPTION
**Pull request by abirembaut**
Since 2022.1-u6 and 2022.2-u3 the following keys can be used as object key in credentials attribute of keycloak-oidc.json
* client-secret-basic (default)
* client-secret-post
* client-secret-jwt or secret-jwt
* private-key-jwt or jwt

Covers [RUNTIME-1650](https://bonitasoft.atlassian.net/browse/RUNTIME-1650)

[RUNTIME-1650]: https://bonitasoft.atlassian.net/browse/RUNTIME-1650?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by laurentleseigneur — APPROVED** [review]
for azure AD, 

```
"credentials": {
    "secret": "{{ azure_client_secret }}"
  }
```

is working, and doesn't requires `client-secret-post` option

**Comment by abirembaut** [comment]
@laurentleseigneur that's thanks to the patch. But with the fix in 2022.2-u3, you will need to change it to:
```
  "credentials" : {
    "client-secret-post": {
      "secret" : "****"
     }
  }
```

**Comment by laurentleseigneur** [comment]
ok, so keep it as it to clearly explain behaviour before/after your fix

File: modules/identity/pages/single-sign-on-with-oidc.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ OpenID Connect configuration is at tenant level.
 [NOTE]
 ====
 Bonita uses http://www.keycloak.org/[Keycloak] OIDC Service client adapter. It has been extended to support other OIDC providers than Keycloak. +
-Authentication in Bonita with OIDC has been tested with both Keycloak server and Google's OpenID Connect endpoint.
+Authentication in Bonita with OIDC has been tested with Keycloak server, Azure AD and Google's OpenID Connect endpoint.
```
Comment (laurentleseigneur) [inline modules/identity/pages/single-sign-on-with-oidc.adoc]: 👍

**Review by vbotnari12 — APPROVED** [review]
All good for release.

**Pull request merged**